### PR TITLE
completed Trello task 2

### PR DESCRIPTION
**Pull request by FlashSampson**
A new README.md document has been created and the old one removed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +21,10 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-games#readme",
   "dependencies": {
-    "pg-format": "^1.0.4",
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "pg": "^8.7.3",
+    "pg-format": "^1.0.4",
+    "supertest": "^6.3.0"
```
Comment (alexdevdavis) [inline package.json]: - Supertest can be installed as a devDependency ... do you need Jest too?

File: README.md [hunk 2]
```diff
@@ -1,57 +1,7 @@
-# Northcoders House of Games API
+# Getting started with this repo
 
-## Background
+To use this repo you'll need create 2 new .env files, .env-development and .env-test.data which shouls contain 'PGDATABASE=' with the relevent database name suffixed as per the example in enc-example file.
```
Comment (alexdevdavis) [inline README.md]: - In production, we wouldn't want to reveal db names, however, for the purposes of this readme and project, what benefit might there be to pointing your users to where they might find the db names?

File: .env-test [hunk 3]
```diff
@@ -0,0 +1 @@
+PGDATABASE=test-data
```
Comment (alexdevdavis) [inline .env-test]: - as per other .env file

File: .env-development [hunk 4]
```diff
@@ -0,0 +1 @@
+PGDATABASE=development-data
```
Comment (alexdevdavis) [inline .env-development]: - These should be gitignored, no? Think you have some punctuation to edit, both in file names and for each database name

**Review by alexdevdavis — CHANGES_REQUESTED** [review]
- Please ensure your .env files aren't being pushed to GitHub, and that your db connections work as intended before moving on to the next endpoint. (as per comments in this pr review)

File: .env-development (hunk 4 above)
Comment (alexdevdavis) [reply .env-development]: - check your db files for what the db names should be, and how they're spelled

Comment (FlashSampson) [reply .env-development]: I have  altered the .env file names to follow the convention laid out in the gitignore folder

File: .env-test (hunk 3 above)
Comment (FlashSampson) [reply .env-test]: I have fixed the file naming convention with this .env file now

File: package.json (hunk 1 above)
Comment (FlashSampson) [reply package.json]: I have installed both supertest and Jest as dev dependencies now

File: README.md (hunk 2 above)
Comment (FlashSampson) [reply README.md]: I have removed the actual database names from the readme as this file isn't gitignored and instead directed users of the repo to the file where the names can be found

**Pull request merged**